.lintstagedrc.json

```diff
@@ -1,5 +1,6 @@
 {
   "linters": {
-    "*.{js,json,css,scss}": ["prettier --write", "git add"]
+    "*.{js,json,css,scss}": ["prettier --write", "git add"],
+    "*.html": ["js-beautify -s 2 -r --html", "git add"]
```

Comment:
One more thing -- you've added js-beautify -- but it is not in devDependencies in package.json -- install it with `npm i --save-dev js-beautify` -- and it will be added

Reply:
oh yessss..